### PR TITLE
Make DatabaseManager more fault tolerant in Cluster

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

We've introduced a couple of fault tolerance improvements into the Grakn Cluster version of DatabaseManager:

- `contains` and `all` will now try to retrieve information from every node, instead of failing if the first node is down
- `create` and `delete` will now be applied to every working node, even if some nodes are down.

For now, this means `create` will no longer throw if the database already exists, and `delete` will no longer throw if the database doesn't exist.

## What are the changes implemented in this PR?

- `contains` and `all` will now try to retrieve information from every node, instead of failing if the first node is down
- `create` and `delete` will now be applied to every working node, even if some nodes are down.

### CODE REVIEW

File: GraknClient.java [hunk 1]
```diff
@@ -38,12 +38,8 @@ static GraknClient core(String address) {
...
     }
 
-    static GraknClient cluster() {
-        return cluster(DEFAULT_ADDRESS);
-    }
-
-    static GraknClient cluster(String address) {
-        return new RPCGraknClientCluster(address);
+    static GraknClient cluster(String... addresses) {
+        return new RPCGraknClientCluster(addresses);
```
Comment (alexjpwalker) [inline GraknClient.java:42]: This change allows Cluster clients to be instantiated with failover.

File: dependencies/graknlabs/repositories.bzl [hunk 2]
```diff
@@ -30,7 +30,7 @@ def graknlabs_common():
     git_repository(
         name = "graknlabs_common",
         remote = "https://github.com/graknlabs/common",
-        tag = "2.0.0-alpha-7" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_common
+        commit = "36564a64904eed7d06d6d7c24055ffc70a40aee8" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_common
```
Comment (alexjpwalker) [inline dependencies/graknlabs/repositories.bzl:33]: This increases the time we wait for Grakn to start, in tests, to 15 seconds.

File: test/behaviour/connection/database/BUILD [hunk 3]
```diff
@@ -41,28 +41,27 @@ java_library(
     ],
 )
 
-grakn_behaviour_java_test(
-    name = "test",
+grakn_java_test(
+    name = "test-core",
```
Comment (alexjpwalker) [inline test/behaviour/connection/database/BUILD:45]: We need to split up this target in order to specify different `@ignore` tags for each.

**Review by lolski — APPROVED** [review]

File: .bazelrc [hunk 4]
```diff
@@ -19,7 +19,7 @@
 
 build --incompatible_strict_action_env --javacopt='--release 8'
 run --incompatible_strict_action_env
-test --incompatible_strict_action_env
+test --incompatible_strict_action_env --test_env=PATH
```
Comment (alexjpwalker) [inline .bazelrc:22]: This change allows Bazel to use the `lsof` utility. Perhaps, in the future, we should change it and include `lsof` in our Bazel runfiles.

**Review by haikalpribadi — APPROVED** [review]

**Pull request merged**